x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -57,63 +96,79 @@ export class HeadlessChromiumDriverFactory {
         throw err;
       }
 
-      safeChildProcess({
-        async kill() {
-          await browser.close();
-        }
-      }, observer);
+      safeChildProcess(
+        {
+          async kill() {
+            await browser.close();
+          },
+        },
+        observer
+      );
 
       // Register with a few useful puppeteer event handlers:
       // https://pptr.dev/#?product=Puppeteer&version=v1.10.0&show=api-event-error
       // https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#class-page
 
       const stderr$ = Rx.fromEvent(page, 'console').pipe(
-        filter(line => line._type === 'error'),
-        map(line => line._text),
+        filter((line: any) => line._type === 'error'),
+        map((line: any) => line._text),
         share()
       );
 
-      const [consoleMessage$, message$] = stderr$.pipe(
-        partition(msg => msg.match(/\[\d+\/\d+.\d+:\w+:CONSOLE\(\d+\)\]/))
-      );
+      const [consoleMessage$, message$] = partition((msg: any) =>
```

Comment:
TS was having issues unravelling how this was supposed to work, so the format change here made it compile properly